### PR TITLE
build(tegel): committing correct stencil -v, igoring stories on build

### PR DESCRIPTION
**Pull request by mJarsater**
**Describe pull-request**  
Set stencil -v to 2.20.0, bumped npm version to latest, ignoring stories files on build.

**Solving issue**  
Fixes: [AB#2845](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2845)

**How to test**  
Try running it locally and run a build. 

**Screenshots**  
-

**Additional context**  
-

### CODE REVIEW

**Review by alexandereneroth — APPROVED** [review]
Nice 🙏

File: tegel/package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
   "module": "dist/index.js",
   "es2015": "dist/esm/index.mjs",
   "es2017": "dist/esm/index.mjs",
-  "types": "dist/types/index.d.ts",
+  "types": "dist/types/components.d.ts",
```
Comment (theJohnnyMe) [inline tegel/package.json:19]: Nice!

File: tegel/package.json [hunk 2]
```diff
@@ -47,7 +47,7 @@
   },
   "dependencies": {
     "@popperjs/core": "^2.11.6",
-    "@stencil/core": "^2.13.0",
+    "@stencil/core": "^2.20.0",
```
Comment (theJohnnyMe) [inline tegel/package.json:50]: Nice!

File: tegel/tsconfig.json [hunk 3]
```diff
@@ -11,5 +11,7 @@
     "jsx": "react",
     "jsxFactory": "h"
   },
-  "include": ["src", "node_modules/@types/jest/index.d.ts"]
+  "include": ["src", "node_modules/@types/jest/index.d.ts"],
+  "exclude": ["**/*.stories.ts", "**/*.stories.js"]
```
Comment (theJohnnyMe) [inline tegel/tsconfig.json:15]: 🤘 awesome!

File: tegel/stencil.config.ts [hunk 4]
```diff
@@ -12,6 +12,7 @@ export const config: Config = {
     },
     {
       type: 'dist-custom-elements',
+      generateTypeDeclarations: true,
```
Comment (theJohnnyMe) [inline tegel/stencil.config.ts:15]: Nice!

**Pull request merged**